mifosng-provider/src/main/java/org/mifosplatform/portfolio/loanaccount/domain/Loan.java

```diff
@@ -802,7 +804,7 @@ private void removeOrModifyTransactionAssociatedWithLoanChargeIfDueAtDisbursemen
     private BigDecimal calculateAmountPercentageAppliedTo(final LoanCharge loanCharge) {
```

Comment:
Please change the method to support for multi disbursement or single disbursement or approved amounts.